tensorrt_llm/_torch/modules/fused_moe/fused_moe_deepgemm.py

```
@@ -377,6 +377,8 @@ class DeepGemmFusedMoE(CutlassFusedMoE):
         model_config (ModelConfig): Configuration object for the model.
     """
 
+    _SUPPORTED_SM_VERSIONS = {100, 103}
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)